samples/sdl/main.c

```diff
@@ -187,7 +186,7 @@ static void
 quit(int rc)
```

Comment:
The parameter is unused now.

Reply:
See existing discussion at https://github.com/XboxDev/nxdk/pull/104#discussion_r281713666 (which used to be resolved by @dracc )